### PR TITLE
policy: document dust threshold for Taproot outputs

### PR DESCRIPTION
**Pull request by darosior**
Following discussions in #22779 .

### CODE REVIEW

**Comment by darosior** [comment]
(Happy to bikeshed on the comment)

**Review by TheBlueMatt — APPROVED** [review]

File: src/policy/policy.cpp [hunk 1]
```diff
@@ -34,6 +34,11 @@ CAmount GetDustThreshold(const CTxOut& txout, const CFeeRate& dustRelayFeeIn)
     int witnessversion = 0;
     std::vector<unsigned char> witnessprogram;
 
+    // Note this computation is for spending a Segwit v0 P2WPKH output (a 33 bytes
+    // public key + an ECDSA signature). For Segwit v1 Taproot outputs the minimum
```
Comment (benthecarman) [inline src/policy/policy.cpp:38]: nit: maybe say `74 byte ECDSA signature`

**Comment by benthecarman** [comment]
ACK d873db7f8ff37c37f9c72482d8ecd52282f4438e

**Comment by ariard** [comment]
Code Review ACK d873db7

**Review by theStack — APPROVED** [review]
ACK d873db7f8ff37c37f9c72482d8ecd52282f4438e

**Pull request merged**